### PR TITLE
Update SHAs for repos not tracked by ProdCon

### PR DESCRIPTION
**Pull request by dseefeld**

### CODE REVIEW

File: tools-local/prebuilt-baseline-offline.xml [hunk 1]
```diff
@@ -136,18 +138,17 @@
     <Usage Id="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <Usage Id="Microsoft.NET.Test.Sdk" Version="15.6.0" />
     <Usage Id="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <Usage Id="Microsoft.NETCore" Version="5.0.0" />
```
Comment (dagood) [inline tools-local/prebuilt-baseline-offline.xml]: Interesting... reeeally old package. Only usage `<AnnotatedUsage Id="Microsoft.NETCore" Version="5.0.0" File="src/vstest/src/Microsoft.TestPlatform.PlatformAbstractions/obj/project.assets.json" Project="src/vstest/" />
`

Comment (dseefeld) [reply tools-local/prebuilt-baseline-offline.xml]: It looks like including uap10.0 as a TargetFramework is pulling this in.  Maybe we should be conditionally removing this for source-build.

**Comment by dseefeld** [comment]
@dotnet-bot test CentOS7.1Release
test CentOS7.1 Unshared Release
test Ubuntu16.04 Release

**Comment by dseefeld** [comment]
test CentOS7.1 Release

**Pull request merged**